src/core/core.scale.js

```diff
@@ -274,51 +296,72 @@ module.exports = Element.extend({
 		me.paddingBottom = 0;
 	},
 	afterSetDimensions: function() {
-		helpers.callback(this.options.afterSetDimensions, [this]);
+		callback(this.options.afterSetDimensions, [this]);
 	},
 
 	// Data limits
 	beforeDataLimits: function() {
-		helpers.callback(this.options.beforeDataLimits, [this]);
+		callback(this.options.beforeDataLimits, [this]);
 	},
 	determineDataLimits: helpers.noop,
 	afterDataLimits: function() {
-		helpers.callback(this.options.afterDataLimits, [this]);
+		callback(this.options.afterDataLimits, [this]);
 	},
 
 	//
 	beforeBuildTicks: function() {
-		helpers.callback(this.options.beforeBuildTicks, [this]);
+		callback(this.options.beforeBuildTicks, [this]);
 	},
 	buildTicks: helpers.noop,
 	afterBuildTicks: function(ticks) {
 		var me = this;
 		// ticks is empty for old axis implementations here
 		if (helpers.isArray(ticks) && ticks.length) {
-			return helpers.callback(me.options.afterBuildTicks, [me, ticks]);
+			return callback(me.options.afterBuildTicks, [me, ticks]);
 		}
 		// Support old implementations (that modified `this.ticks` directly in buildTicks)
-		me.ticks = helpers.callback(me.options.afterBuildTicks, [me, me.ticks]) || me.ticks;
+		// eslint-disable-next-line callback-return
+		me.ticks = callback(me.options.afterBuildTicks, [me, me.ticks]) || me.ticks;
 		return ticks;
 	},
 
 	beforeTickToLabelConversion: function() {
-		helpers.callback(this.options.beforeTickToLabelConversion, [this]);
+		callback(this.options.beforeTickToLabelConversion, [this]);
 	},
-	convertTicksToLabels: function() {
+	convertTicksToLabels: function(ticks) {
```

Comment:
Are we worried about changing this method signature? E.g. if someone was calling it with no args and expecting it to use `me.ticks` then that might not work anymore

Reply:
The old behavior is handled in `else` case